ValidationFactory/Validators/ErrorCode.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ValidationFactory.Validators
+{
+    internal class ErrorCode
+    {
+        public const string InvalidStagingId = "StagingId must be valid non-negative identifiers >= 1.";
+
+        public const string InvalidCounterpartyId = "CounterpartyId must be valid non-negative identifiers >= 1.";
+
+        public const string InvalidPrincipalId = "PrincipalId must be valid non-negative identifiers >= 1.";
+
+        public const string InvalidSourceAccountId = "SourceAccountId must be valid non-negative identifiers >= 1.";
+
+        public const string InvalidCurrentPrice = "CurrentPrice should be a valid price: >= 0 and have only 2 digits after delimiter.";
+
+        public const string InvalidIsin = "Isin should be a valid ISIN. It should start with 2 capital letters and then 10 digits.";
```

Comment:
We could use resx file for this, but because the task was enormously large, I decided that constants also works for this homework.